core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -354,20 +364,20 @@ async function prepareUpgradeCalldata(
     mainContract: zksync.types.Address,
     params: {
         l2ProtocolUpgradeTx: {
-            txType: BigNumberish;
-            from: BigNumberish;
-            to: BigNumberish;
-            gasLimit: BigNumberish;
-            gasPerPubdataByteLimit: BigNumberish;
-            maxFeePerGas: BigNumberish;
-            maxPriorityFeePerGas: BigNumberish;
-            paymaster: BigNumberish;
-            nonce?: BigNumberish;
-            value: BigNumberish;
-            reserved: [BigNumberish, BigNumberish, BigNumberish, BigNumberish];
+            txType: ethers.BigNumberish;
+            from: ethers.BigNumberish;
+            to: ethers.BigNumberish;
+            gasLimit: ethers.BigNumberish;
+            gasPerPubdataByteLimit: ethers.BigNumberish;
+            maxFeePerGas: ethers.BigNumberish;
+            maxPriorityFeePerGas: ethers.BigNumberish;
+            paymaster: ethers.BigNumberish;
```

Comment:
it might make sense to import BigNumberish directly still if we use it so much, wdyt?

Reply:
In case of `BigNumberish` it probably makes sense.

Reply:
updated